### ISSUE TITLE
No corresponding symop found

### ISSUE THREAD

**Issue by antoine-levitt** [JuliaMolSim/DFTK.jl#113]
Test with silicon.jl and
```
kgrid = [1, 4, 4]        # k-Point grid
supercell = [2, 1, 1]    # Lattice supercell
```

**Comment by antoine-levitt** [comment]
and with `kgrid=[1,2,2]` I get a bounds error (although that might be just on the FFTs branch)

**Comment by mfherbst** [comment]
Hmmm ... I had those a few times as well. But gut that you found a simple case to reproduce the issue. I'll look into it.

**Comment by mfherbst** [comment]
I cannot reproduce the `[1, 2, 2]` bug and actually get the right numbers, so I suppose that's fixed.

**Comment by mfherbst** [comment]
For the `[2,1,1]` I think it is related to the last paragraph of https://atztogo.github.io/spglib/api.html#spg-get-symmetry. I'm investigating.

**Comment by antoine-levitt** [comment]
Might be good to add tests for weird kpoint grids (eg these shifted grids people use, if spglib exposes them)

**Comment by mfherbst** [comment]
Shifted grids are not yet implemented in DFTK (but in spglib), but I think that's one possible way to go to tackle this problem, actually.

Tests for sure, but we first need to get it to work ;).

**Comment by mfherbst** [comment]
This is fixed with spglib 1.14.1 and #125.
